src/index.js

```diff
@@ -69,17 +69,26 @@ export default class Machine extends erx.Bus<State> {
   }
 
   send(event: Event, data: any): boolean {
-    if (!this.state || !this.states[this.state] || this.transitioning) {
+    if (!this.state || this.transitioning) {
       return false;
```

Comment:
Should this throw?

Reply:
Depends how strict we want to be. I think cyan/fsm has been built so far with few errors, and not to throw unless it actually breaks the adunit? Could be worth thinking about if we start logging/tracking adunits failing like Byron said today.

Reply:
It should throw, if we want to prevent people to actually meet these conditions? But I guess it should never throw if "this.transitioning" is true? Because it just means it's between states

Reply:
We might not need this.transitioning depending on how the transition function chain is written but yeah, if transitioning then just ignore. I could add a throw for if(!this.state) .... but like i'm not even sure how this.state could be undefined at this point....i'll add the throw just in case, but hopefully we can remove that check.